build.sbt

```diff
@@ -219,12 +219,8 @@ lazy val unit = project
 
       put(
         "inputClasspath",
-        fullClasspath.in(testsInput, Compile).value.map(_.data)
-      )
-
-      put(
-        "semanticdbRootDirectory",
-        Seq(semanticdbTargetRoot.in(testsInput, Compile).value)
+        fullClasspath.in(testsInput, Compile).value.map(_.data) :+
+          semanticdbTargetRoot.in(testsInput, Compile).value
```

Comment:
similar strategy as in https://github.com/scalacenter/sbt-scalafix/commit/cd9efb1b1e16f178da6be72a4cc621524e8a301a